examples/URL-Scrapping-RAG/README.md

```diff
@@ -0,0 +1,13 @@
+## URL Scrapping RAG
+
+### Overview
+This repository contains code and implementation for URL Scrapping RAG. 
```

Comment:
change the readme stating what this example will include and how it will work with flow in bullet points to make it easy to understand